ports/physac/portfile.cmake

```
@@ -0,0 +1,14 @@
+#header-only library
+set(VCPKG_BUILD_TYPE release)
+
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO victorfisac/Physac
+    REF "${VERSION}"
+    SHA512 c539ee73d6f456e592d4a92cc5707278476632626b0fa0edfe6396cd4460fe0c2669843f4df3a22a132664d1981d261601061cca76ad1e4b63510a901fc3987b
+    HEAD_REF master
+)
+
+file(INSTALL "${SOURCE_PATH}/src/physac.h"  DESTINATION "${CURRENT_PACKAGES_DIR}/include")
+
+vcpkg_install_copyright(FILE_LIST "${CMAKE_CURRENT_LIST_DIR}/LICENSE")
```

```suggestion
vcpkg_install_copyright(FILE_LIST "${SOURCE_PATH}/LICENSE")
```

Why did you copy the license file here rather than getting it from the source distribution?

---

(I will make this change for you if you want but this was sufficiently unusual I wanted to understand why you do that :) Thanks for the new port!)

---

The currently used tag is `1.1`, which is generated at Nov 3, 2019, but the `LICENSE file` of upstream is added Jul 24, 2022, so `LICENSE file` does not exist when downloading 1.1 source.

https://github.com/victorfisac/Physac

---

If we want to use `vcpkg_install_copyright(FILE_LIST "${SOURCE_PATH}/LICENSE")`, then we need to download the `latest commit` of upstream instead of the `1.1` tag.

---

Right, I copied the license file here rather than getting it from the source distribution because version 1.1 does not include license file.

---

Ah, I missed that that was changed. Thanks for clarifying!